Added suppressConstructorProperties = true

donovanmuller opened the pull request:
Without adding `suppressConstructorProperties = true` to `@AllArgsConstructor` I hit this issue: http://stackoverflow.com/q/39381474/2408961.

I'm not sure why I get this now with the `0.2` release and not whilst testing ¯\\_(ツ)_/¯

zanella commented:
Hey, before I accept the PR can you just try out the release 2.8.4 of jackson ? (https://github.com/FasterXML/jackson-databind/issues/1239#issuecomment-250634845) maybe it magically does? otherwise we just add the "suppress"...

donovanmuller commented:
I'll check it a bit later and update...

donovanmuller commented:
@zanella Updated to 2.8.4, reverted back to just using `@AllArgsConstructor` and still get the issue. 

Added `@AllArgsConstructor(suppressConstructorProperties = true)` and it works again.
